### PR TITLE
docs: comments for template examples in simple.yml

### PR DESCRIPTION
**Pull request by base698**

### CODE REVIEW

File: doc/examples/simple.yml [hunk 1]
```diff
@@ -10,6 +10,8 @@ global:
   hipchat_api_url: 'https://hipchat.foobar.org/'
 
 # The directory from which notification templates are read.
+#   See: https://prometheus.io/docs/alerting/notification_examples/#defining-reusable-templates
```
Comment (brian-brazil) [inline doc/examples/simple.yml]: This is fairly advanced stuff, I'm not sure we should be pointing users to here from a very basic example.

Comment (simonpasquier) [reply doc/examples/simple.yml]: I agree.

Comment (base698) [reply doc/examples/simple.yml]: This references a template but has no corresponding example.  It was confusing when I was trying to find a way to update the text in a notification.  Maybe take it out or provide a complete example?

Comment (brian-brazil) [reply doc/examples/simple.yml]: Removing it is probably best.

**Pull request closed without merging**